src/utils/storage.ts

```
@@ -0,0 +1,126 @@
+    return new Promise((resolve) => {
+      chrome.storage.local.remove([STORAGE_KEYS.CONVERSATIONS, STORAGE_KEYS.ACTIVE_CONVERSATION], () => {
+        resolve();
+      });
+    });
+  }
+
+  static async getConversationList(): Promise<Conversation[]> {
+    const storage = await this.getAll();
+    return Object.values(storage.conversations).sort((a, b) => b.updatedAt - a.updatedAt);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)